packetbeat/protos/redis/redis_parse.go

```diff
@@ -423,7 +424,7 @@ func (p *parser) parseArray(depth int, buf *streambuf.Buffer) (common.NetString,
 	}
 
 	// handle top-level request command
-	if depth == 0 && isRedisCommand(content[0]) {
+	if depth == 0 && (isRedisCommand(content[0]) || isRedisCommand(bytes.Join(content[0:2], []byte(" ")))) {
```

Comment:
I think this needs a check in the (unlikely?) case that `count < 2`

Reply:
Would you explain more details about what to check? I just don't understand what do you mean.

Reply:
I mean accessing `content[0:2]` can fail if `len(content)` is 1, so you can write
```
 	if depth == 0 && (isRedisCommand(content[0]) || (count > 1 && isRedisCommand(bytes.Join(content[0:2], []byte(" "))))) { 
```

Reply:
You are right. I get it.